### DISCUSSION TITLE
Include non md files in the Navigation menus

### DISCUSSION THREAD

**Discussion by 73DX** [squidfunk/mkdocs-material#2232] [Q&A] [answered]
Hello, 
I wonder if non markdown files can be included in the left navigation menu.
We would like to include files that are not Markdown files and have then automatically listed under each corresponding folder.
Is that possible?
Thanks,
Martin

**Comment by squidfunk** [comment]
You can add URLs to the `nav` in `mkdocs.yml`

**Reply by 73DX** [reply]
Hi Martin, 
My apologies, I did not describe the idea well enough.
What I meant is if there is a way for Mkdocs to automatically generate those links.
Thanks.

**Reply by squidfunk** [reply]
AFAIK, MkDocs will only auto-include `*.md` files when you don't specify the `nav` entry, not other files.

**Comment by markallasread** [comment, marked as answer]
In case anyone encounters this, mkdocs has a light article on it: https://www.mkdocs.org/user-guide/writing-your-docs/#configure-pages-and-navigation

As noted by squidfunk it will not generate links to any other files listed.
